source/elements/oneDAL/source/common_interface/header_files_and_namespaces.rst

```diff
@@ -16,18 +16,38 @@ oneDAL public identifiers are represented in the following header files:
      - A header file for a particular algorithm.
        The folder for the algorithm itself is ``oneapi/dal/algo/%ALGO%/``.
 
-       The string %ALGO% should be substituted with the name of the algorithm, for example, ``kmeans`` or ``knn``.
+       The string %ALGO% should be substituted with the name of the algorithm,
+       for example, ``kmeans`` or ``knn``.
 
    * - ``oneapi/dal/algo/misc/%FUNC%.hpp``
-     - A header file for miscellaneous data types and functionality that is intended to be used by oneDAL algorithms and applications
+     - A header file for miscellaneous data types and functionality that is
+       intended to be used by oneDAL algorithms and applications
        of the analytical pipeline.
 
-       The string %FUNC% should be substituted with the functionality name, for example, ``mt19937`` or ``cross_enthropy_loss``.
+       The string %FUNC% should be substituted with the functionality name, for
+       example, ``mt19937`` or ``cross_enthropy_loss``.
+
+   * - ``oneapi/dal/table/%FILE%.hpp``
+     - A header file for types and entities related to the :txtref:`table` concept.
```

Comment:
what is the difference between 'types' and 'entities'?

Reply:
No difference. Shorten just for 'types'